src/components/CodeEditor.js

```diff
@@ -169,11 +195,22 @@ class CodeEditor extends Component {
   componentWillReceiveProps (nextProps) {
     // If the incoming game is the empty game code,
     // set CodeMirror's value to ''.
+    let lesson;
+    let slide;
+    if(nextProps.tutorial){
+      lesson = lessons[nextProps.tutorial.lessonIndex]
+      const { slides } = lesson
+      slide = slides[nextProps.tutorial.slideIndex]
+    }
     if (nextProps.game === 'SCRIPT-8 NEW') {
       this.setContents(blank)
     }
     if (nextProps.game.startsWith('SCRIPT-8 LESSON')) {
       this.setContents(nextProps.game.replace('SCRIPT-8 LESSON', ''))
+
+      if(slide && slide.lineToFoucs){
```

Comment:
Should lineToFoucs be lineToFocus?

Reply:
Ugh😭😭! Sorry about that, I will update when I get back my computer